### ISSUE TITLE
IFTPServer.Status Never Resets

### ISSUE THREAD

**Issue by Taullarcis** [FubarDevelopment/FtpServer#137]
I am using this project in a way that needs to allow for modification of the IP, port, and authentication without stopping the application and I am needing to avoid using multiple ServiceCollections.

As it currently stands this code is dependent upon using its own ServiceCollection to allow for this to function as  once the Stop() or StopAsync() method are called the IFTPServer.Status never reverts back to a state that will allow the server to be run again.

Would it be possible to implement some way that will allow for the server to be updated and reset without requiring a rebuilt service collection?

**Comment by fubar-coder** [comment]
No, it's not possible (yet) due to the way the source code is currently designed. I'll leave this issue open as a reminder for future development.
